docs/grpc/docs.md

```diff
@@ -3807,6 +3813,110 @@ For example, if `oversampling` is 2.4 and `limit` is 100, then 240 vectors will
 
 
 
+<a name="qdrant-SearchMatrixOffsets"></a>
+
+### SearchMatrixOffsets
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| offsets_row | [uint64](#uint64) | repeated | Row coordinates of the CRS matrix |
```

Comment:
After some investigation I realized that CRS stands for something else. What we are doing with offsets is more similar to the COO representation

Reply:
Good observation :+1: 
I will get this fixed.